web/source/kmwbase.ts

```diff
@@ -548,6 +548,10 @@ namespace com.keyman {
     ['activatingUI'](state: boolean) {
       this.uiManager.setActivatingUI(state);
     } 
+
+    // Functions that might be added later
+    ['beepKeyboard']: () => void;
+    ['oninserttext']: (dn: number, s: string) => void;
```

Comment:
Can these get added on a separate PR (when the functions get implemented)?

Reply:
No.  They result in TypeScript errors otherwise, since they get referenced within KMW code.  (It thinks they don't exist and are thus uncallable, despite checking for the definition.)